### PR TITLE
Better IntelliJ debugging support.

### PR DESCRIPTION
**Pull request by kshakir**
Missing *-version.conf files will no longer cause errors when running from IntelliJ.
Fixed projectName (ex: 'cromwell-engine') vs artifactName (ex: 'cromwell-engine.jar').
Also snuck in sbt 1.x syntax fix missed in a prior commit.

### CODE REVIEW

File: common/src/main/scala/common/util/VersionUtil.scala [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    s"${versionConf(projectName)}-to-be-generated-by-sbt"
+  }
+
+  private val SbtDependencyVersionRegex = """\s+val (\w+)V = "([^"]+)"""".r
+
+  def sbtDependencyVersion(dependencyName: String)(projectName: String): String = {
+    try {
+      val dependencies = Paths.get("project/Dependencies.scala").toAbsolutePath
+      val lines = Files.readAllLines(dependencies).asScala
+      val versionMatch = lines collect {
```
Comment (Horneth) [inline common/src/main/scala/common/util/VersionUtil.scala]: `collectFirst` and you can get rid of the `headOption` below

File: engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala [hunk 2]
```diff
@@ -578,7 +580,8 @@ object CromwellApiServiceSpec {
       case ValidateWorkflowId(id) =>
         if (RecognizedWorkflowIds.contains(id)) sender ! MetadataService.RecognizedWorkflowId
         else sender ! MetadataService.UnrecognizedWorkflowId
-      case GetCurrentStatus => sender ! StatusCheckResponse(true, Map("Engine Database" -> SubsystemStatus(true, None)))
+      case GetCurrentStatus => sender !
+        StatusCheckResponse(ok = true, systems = Map("Engine Database" -> SubsystemStatus(ok = true, messages = None)))
```
Comment (Horneth) [inline engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala]: ToL: can we have `sender !` on the new line too ?

File: project/Settings.scala [hunk 3]
```diff
@@ -142,13 +142,13 @@ object Settings {
       // The assembly task generates a fat JAR file
       val artifact: File = assembly.value
       val artifactTargetPath = s"/app/${artifact.name}"
-      val artifactName = name.value
+      val projectName = name.value
```
Comment (mcovarr) [inline project/Settings.scala]: oops

**Comment by geoffjentry** [comment]
:+1:

**Pull request merged**